views/prisonerProfile/partials/prisonerProfileHeader.njk

```diff
@@ -172,4 +185,23 @@
             {% endif %}
         </div>
     </div>
-</div>
\ No newline at end of file
+</div>
+
+{% block pageScripts %}
+    <script>
+      function expandLink(href) {
```

Comment:
Should this JS not be added to the `education-actions.js` file?

Reply:
Moving, but the use of `prisoner-accordion-heading-5` is not acceptable going forward.

Mons has agreed to revisit in the near future, as all this `gtag` work needs reviewing anyway